pkg/cmd/dev/cache.go

```diff
@@ -131,6 +131,7 @@ func (d *dev) cacheIsUp(ctx context.Context) bool {
 // string which is a line that should be added to ~/.bazelrc.
 func (d *dev) setUpCache(ctx context.Context) (string, error) {
 	if d.cacheIsUp(ctx) {
+		log.Printf("Cache is already up\n")
```

Comment:
This seems like a noisy thing to print on every `dev` invocation. Can you remove this line?

Reply:
Done.